internal/schema/testdata/ciBranchPicky.json

```diff
@@ -11,6 +11,25 @@
       "storageCalc": 1,
       "openshiftProjectPattern": null,
       "developmentEnvironmentsLimit": 5,
+      "deployTargetConfigs": [
+        {
+      "branches": "qa",
+      "deployTarget": {
+      "id": 157,
+      "name": "test.amazee.io",
+      "token": "test"  },
+      "id": 2094,
+      "project": {
+      "autoIdle": 1,
+      "gitUrl": "ssh://git@bitbucket.biscrum.com:7999/wpsites/test.git",
+      "name": "test-template",
+      "productionEnvironment": "master",
+      "storageCalc": 1
+                  },
+      "pullrequests": "false",
+      "weight": 102
+            }
+      ],
```

Comment:
Fixing the indentations on this would be nice for readability

Reply:
All of the .json files could benefit from some formatting. Maybe run them through https://jsonlint.com/